lib/bunyan.js

```diff
@@ -1008,7 +1008,7 @@ Logger.stdSerializers.req = function req(req) {
         return req;
     return {
         method: req.method,
-        url: req.url,
+        url: req.originalUrl,
```

Comment:
This should be `url: req.originalUrl || req.url,` to maintain compatibility with non express(4.x) users.